project/resale_prices.js

```diff
@@ -0,0 +1,23 @@
+import util from './utils.js'
+
+export default function(xAxis,yAxis, keyword) {
```

Comment:
This function should take the full dataset, district and flat type as arguments. Data filtering should happen here, because when we load the full dataset (as the user loads the page), we don't have the district and flat type information yet. It comes later. Allowing district and flat type to be passed in as argument gives us the flexibility to invoke this function when we are ready with all the necessary information.

Reply:
Done